libr/core/cmd_print.c

```diff
@@ -2393,10 +2393,11 @@ static void cmd_print_pv(RCore *core, const char *input, const ut8* block) {
 		// r_num_get is gonna use a dangling pointer since the internal
 		// token that RNum holds ([$$]) has been already freed by r_core_cmd_str
 		// r_num_math reload a new token so the dangling pointer is gone
-		r_cons_printf ("{\"value\":%"PFMT64u ",\"string\":\"%s\"}\n",
-			r_num_math (core->num, "[$$]"),
-			str
-			);
+		RPrintJSON *j = r_print_json_begin ('{', r_cons_printf);
```

Comment:
Like `ojc_fill` in http://www.ohler.com/ojc/doc/index.html

Reply:
its not the same

Reply:
I think we should have a consistent format, which means whatever a command returns it should always be an object.
So specifying `{` is useless because it will be an object anyway.
Or maybe add an API such as `r_printf_json_begin_object` but it's a bit verbose.

Reply:
the idea to pass { is to make it flexible for [ too, but yeah as long as there are only 2 types of blocks having separate functions makes sense, the _end doesnt require to specify the type because its implicitly popped from the stack